.goreleaser.yml

```diff
@@ -45,10 +45,15 @@ builds:
 
 archives:
   - format: binary
-    replacements:
```

Comment:
`replacements` was deprecated.
To upgrade GoReleaser, we need to fix this.
https://goreleaser.com/deprecations/?h=replacements#__tabbed_14_2